game_event/templates/game-events.html

```diff
@@ -39,6 +39,7 @@ <h1>Game Events</h1>
 				<th>Ballgame</th>
 			</tr>
 		</thead>
+        <a href="/game-events/create" class="btn btn-primary mb-2">Create Game Event</a>
```

Comment:
Do we have a test already that verifies this url?
Something similar to this one:
```
    def test_loading_site_no_filters(self, client):
        response = client.get('/game-events/')
        assert response.status_code == 200
```

Reply:
Yea in meet_balls_app.test I had a test that checks this url